### PR TITLE
Feature/degraded access no login

### PR DESCRIPTION
**Pull request by ahaurw01**
Addresses 2 of 3 bullet points from #40 - accessing the lobby and games if you are not authenticated.

### CODE REVIEW

File: app/controllers/chat/channel.js [hunk 1]
```diff
@@ -1,6 +1,10 @@
 import Ember from 'ember';
 
 export default Ember.Controller.extend({
+  canParticipate: Ember.computed.readOnly('session.isAuthenticated'),
+
+  cannotParticipate: Ember.computed.not('canParticipate'),
```
Comment (SirZach) [inline app/controllers/chat/channel.js]: what do you think about this being a computed property on the session itself? maybe like a <code>hasWriteAccess</code>?

Comment (ahaurw01) [inline app/controllers/chat/channel.js]: Sure, sounds good.

**Comment by ahaurw01** [comment]
Updated - easy enough now to call on `session.hasWriteAccess` or `session.doesNotHaveWriteAccess`.

**Comment by SirZach** [comment]
Nice!

**Comment by ahaurw01** [comment]
Woot!

**Pull request merged**